types/fs-extra/index.d.ts

```diff
@@ -304,6 +230,22 @@ export interface ReadOptions {
     flag?: string;
 }
 
+export interface RemoveOptions {
+    maxBusyTries?: number;
+    unlink?: typeof fs.unlink | typeof unlink;
```

Comment:
My implementation may not be the best. The functions have to be the union of the callback and promise types.